src/commands/login.ts

```diff
@@ -126,12 +116,29 @@ export const login = async (
   if (!options?.skipAuthenticate) {
     const authn = await authenticate({ debug: options?.debug });
     await validateTenantAccess(authn, options?.debug);
-  }
 
-  if (!loggedIn) {
-    print2(
-      `You are now logged in to the ${orgSlug} organization, and can use the p0 CLI.`
-    );
+    if (!loggedIn) {
+      print2(
+        `You are now logged in to the ${orgSlug} organization, and can use the p0 CLI.`
+      );
+    } else {
+      print2(`You are currently logged in to the ${orgSlug} organization.`);
+    }
+
+    if (tokenTimeRemaining >= 0) {
```

Comment:
completely when we create a new login we don't properly update the final time and end up saying: "The current session expires in 0h00m00s"